backend/poaps/services.py

```
@@ -71,6 +71,11 @@ def decrypt_token(ciphertext):
 def validate_drop_capacity(drop):
     if drop.status != PoapDrop.STATUS_ACTIVE:
         raise ClaimClosedError('This POAP is not open for claiming.')
+    now = timezone.now()
+    if drop.event_start_at and now < drop.event_start_at:
+        raise ClaimClosedError('This POAP is not open for claiming yet.')
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)